lib/language_pack/base.rb

```diff
@@ -10,6 +10,7 @@
 require "language_pack/instrument"
 
 Encoding.default_external = Encoding::UTF_8 if defined?(Encoding)
+ENV["BPLOG_PREFIX"] ||= "buildpack.ruby"
```

Comment:
IIUC this won't set the prefix if it's already set, but is that what we want? Won't this be set if another buildpack has run before this?

Reply:
No because buildpacks run with isolated environments. Only the export feature can be used to forward e.g. `$PATH` for following buildpacks.

Reply:
Unless you're putting it in the export folder/file directory thing that gets run between multi-buildpack builds it shouldn't be picked up by another buildpacks value.

That being said i'm fine we we want to hardcode this value to over-write what's already there I could do that, I just didn't know if there was ever a future where it could be modified by codon/etc.

Reply:
Why would you want this to be configurable?

Reply:
🤷‍♂️ I originally thought the value was being set via codon/cytokine-thing-i-cannot-spell until this PR.

Reply:
I updated to not be `||=`